tests/plotting/test_collection.py

```diff
@@ -3,12 +3,18 @@
 import weakref
 
 import numpy as np
+import pytest
 import vtk
 
 import pyvista as pv
 from pyvista.core._vtk_core import vtk_to_numpy
 
 
+@pytest.fixture(autouse=True)
+def validate_gc(check_gc):
+    pass
```

Comment:
Do I understand correctly that the 3 copies of this fixture just serve to trigger autouse on the non-autouse `check_gc` fixture? Might be worth a comment or docstring.

Reply:
Yes. I've added a docstring to clarify this. These three modules were the original only three under `tests/plotting/` where these GC fixtures were autoused. We should come back to this later to enable the fixture to be used for the entire module, but this requires code fixes to get clean up references.

Reply:
Future looking comment, i.e. not to address in this PR, if the goal is to have `check_gc` apply to as many tests as possible, then it will be better to have `check_gc` be `autouse=True` and then provide a mechanism to override this.  This will require a slight rework of `check_gc` to expose a method by which to skip checking in a yielded object, similar to how we do in `validate_image_cache`.  The current state is good for this PR.